pages.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // ConvertPages converts a Pages file to text.
-func ConvertPages(r io.Reader) (string, map[string]string, error) {
+func ConvertPages(r io.Reader, readability bool) (string, map[string]string, error) {
```

Comment:
@Ravf95 this is a backwards incompatible change. Instead, we might have to explorer a solution like in my comment here: https://github.com/sajari/docconv/pull/75/files#r449309398